packages/balancer/skills/liquidity_provision_abci/models.py

```diff
@@ -71,3 +71,14 @@ def _ensure_managed_pool_address(  # pylint: disable=no-self-use
             "'managed_pool_address' is a required parameter",
         )
         return managed_pool_address
+
+    def _ensure_multisend_address(  # pylint: disable=no-self-use
+        self, kwargs: Dict
+    ) -> str:
+        """Ensure that the MultiSend address is available."""
+        multisend_address = kwargs.get("multisend_address", None)
+        enforce(
+            multisend_address is not None,
+            "'multisend_address' is a required parameter",
+        )
+        return multisend_address
```

Comment:
@DavidMinarsch this had to be added because multisend_address is necessary both in TerminationParams and LPParams. And since _ensure() utilizes kwargs.pop only one of them can make use of it.

Reply:
ok - another thing to monitor.